internal/plugins/project-quota.go

```diff
@@ -193,7 +193,7 @@ func (m *assetManagerProjectQuota) getQuotaStatus(assetType db.AssetType, projec
 		return nil, err
 	}
 	if project == nil {
-		return nil, fmt.Errorf("project not found in Keystone: %s", projectID)
+		return nil, core.AssetNotFoundErr{InnerError: fmt.Errorf("project not found in Keystone: %s", projectID)}
```

Comment:
Invoking the special behavior of `AssetNotFoundErr` here is pointless. Since `ListAssets` always returns the project ID, even if the project was deleted, this will just lead us into a loop of deleting and recreating the asset. I'd rather have the error bubble up into our alerting so that we can see that the project resource needs to be cleaned up.